app/src/main/res/values/strings.xml

```diff
@@ -404,4 +404,13 @@
     <string name="metric_newest">\"SUSI, what are the newest skills?\"</string>
     <string name="metrics_top_games">\"SUSI, what are your top games?\"</string>
     <string name="message_no_skills_found">No skills found.</string>
+
+    <!--Report Skill -->
+
+    <string name="report_skill">Report skill</string>
+    <string name="report_send">Send report</string>
+    <string name="report_error">Report skill unsuccessful </string>
```

Comment:
Why an extra space at the end of the string?

Reply:
Please change the string.
Change it to something like this "Error reporting skill. Please try again later"